rtgui/adjuster.cc

```diff
@@ -25,93 +25,106 @@
 #include "guiutils.h"
 #include "rtimage.h"
 
-#define MIN_RESET_BUTTON_HEIGHT 17
 
-static double one2one(double val)
+namespace {
+
+constexpr int MIN_RESET_BUTTON_HEIGHT = 17;
+
+double one2one(double val)
 {
     return val;
 }
+}
 
-Adjuster::Adjuster (Glib::ustring vlabel, double vmin, double vmax, double vstep, double vdefault, Gtk::Image *imgIcon1, Gtk::Image *imgIcon2, double2double_fun slider2value_, double2double_fun value2slider_)
-{
+Adjuster::Adjuster (
+    Glib::ustring vlabel,
+    double vmin,
+    double vmax,
+    double vstep,
+    double vdefault,
+    Gtk::Image *imgIcon1,
+    Gtk::Image *imgIcon2,
+    double2double_fun slider2value,
+    double2double_fun value2slider)
```

Comment:
You asked me about this. I usually break it this way, but yours is fine, too:

### C'tor
```c++
Class::Class(
    const Type& _param1,
    const Type& _param2,
    Enum _param3
) :
    param1(_param1),
    param2(_param2),
    param3(_param3)
{
}
```

### Function / method
```c++
ReturnType getReturnType(
    const Type& _param1,
    const Type& _param2,
    Enum _param3
)
{
    return {};
}
```